fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/ComposeTopSitesRobot.kt

```diff
@@ -110,15 +106,12 @@ class ComposeTopSitesRobot(private val composeTestRule: HomeActivityComposeTestR
             interact: ComposeTopSitesRobot.() -> Unit,
         ): Transition {
             composeTestRule.contextMenuItemRename().performClick()
-            Espresso.onView(
-                Matchers.allOf(
-                    ViewMatchers.withId(R.id.top_site_title),
-                    CoreMatchers.instanceOf(EditText::class.java),
-                ),
-            )
-                .perform(ViewActions.replaceText(title))
-            Espresso.onView(ViewMatchers.withId(android.R.id.button1))
-                .perform((ViewActions.click()))
+            itemWithResId("$packageName:id/top_site_title")
```

Comment:
I see there already is a composeTestRule.topSiteItem(title). Could we reuse that here?

Reply:
That refers to the top site title from the top sites list. (is composable)

This refers to the text box from the "Rename" dialog. (is not composable)